content/en/docs/Getting started/_index.md

```diff
@@ -98,11 +148,54 @@ Specify the [Docsy theme](https://github.com/google/docsy) like any other Hugo t
     $ git submodule update --init --recursive
 
 
-You can either use the theme as a submodule like the example site, as shown above (our recommended approach for easy updates), or can just [clone the theme](https://gohugo.io/themes/installing-and-using-themes/#install-a-single-theme) into your project's `themes` subdirectory. For simplicity we recommend copying and editing our [example site configuration](#configuring-your-site) for your project or you may get Hugo errors for missing parameters and values when you try to build your site.
+You can either use the theme as a submodule like the example site, as shown above (our recommended approach for easy updates), or can just [clone the theme](https://gohugo.io/themes/installing-and-using-themes/#install-a-single-theme) into your project's `themes` subdirectory. For simplicity we recommend copying and editing our [example site configuration](#configuring-your-site) for your project, or you may get Hugo errors for missing parameters and values when you try to build your site.
```

Comment:
Think we can remove this para as it's pretty much all said in the intro to this section.

Reply:
done

Reply:
done